ansible/roles/blackbox_exporter/templates/blackbox_exporter.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Prometheus Blackbox Exporter
+
+[Service]
+User=nobody
+Group=nogroup
+EnvironmentFile=-/etc/default/blackbox_exporter
+PermissionsStartOnly=true
+ExecStartPre=/sbin/setcap 'cap_net_raw+ep' {{ blackbox_exporter_path }}/blackbox_exporter
```

Comment:
Why does it need `cap_net_raw`? I **assume**, it's for ICMP PING, but it can be done with SOCK_DGRAM, IPPROTO_ICMP as well.